README.md

```diff
@@ -57,6 +57,33 @@ nix-env -iA cachix -f https://cachix.org/api/v1/install
 cachix use fencer
 ```
 
+## Environment variables
+
+There are environment variables that can be optionally set, which can
+have an effect on Fencer execution.
+
+Logging environment variables are:
+
+- `LOG_BUFFER` - The logging buffer size in bytes.
+- `LOG_LEVEL` - The logging level. It can be one of the following:
```

Comment:
This should say what the default is. Same for `LOG_NETSTR`.

Reply:
I addressed this with https://github.com/juspay/fencer/pull/65.